code_challenges/advent_of_code/repetition_code.py

```diff
@@ -0,0 +1,50 @@
+def get_messages():
+    with open("./code_challenges/advent_of_code/message_input") as message_input:
+        messages = message_input.read().split("\n")
+    return messages
+
+
+def break_up_messages():
+    messages = get_messages()
+    character_breakdown = {}
+    for i in range(len(messages[0])):
+        character_breakdown[f"position {str(i)}"] = {}
+
+    for message in messages:
+        for i, character in enumerate(message):
+            character_breakdown[f"position {str(i)}"][character] = (
+                character_breakdown[f"position {str(i)}"].get(character, 0) + 1
+            )
+    return character_breakdown
+
+
+def find_hidden_message_high():
+    character_breakdown = break_up_messages()
+
+    hidden_message = ""
+    for position, characters in character_breakdown.items():
```

Comment:
position isn't used because you are using a dictionary as a list.

Reply:
Also, when you iterate over this dictionary, you are relying on it being in a certain order.  This isn't a dictionary thing even though it's working in this case.